app/src/interfaces/select-color/select-color.vue

```diff
@@ -361,6 +367,15 @@ function useColor() {
 			emit('input', getHexa());
 		}
 	}
+
+	function roundColorValues(arr: number[]): number[] {
+		if (arr.length === 4) {
+			// do not round the opacity
```

Comment:
```suggestion
			// Do not round the opacity
```

(Do we need https://eslint.org/docs/latest/rules/capitalized-comments ? 😄)

Reply:
We very much do if thats a thing ppl care about 😂

Reply:
> We very much do if thats a thing ppl care about 😂

I'll probably indeed come up with a pull request to do so 😄 Gut feeling says 90% of the comments already are capitalized, so it would be nice if we had full consistency (and a clear rule) for this as well 👍